ddtrace/internal/datadog/profiling/dd_wrapper/include/uploader_builder.hpp

```diff
@@ -25,6 +25,7 @@ class UploaderBuilder
     static inline std::string profiler_version;
     static inline std::string url{ "http://localhost:8126" };
     static inline ExporterTagset user_tags{};
+    static inline std::string output_filename{ "" };
```

Comment:
Why static?

Reply:
This actually is a output_filename_prefix and it's followed by `<pid>.<counter>` where counter is incremented for each `UploaderBuilder::build` call.

Reply:
Chatted offline and agree that static member variables in here need not be static. Will follow up in a separate PR to revisit this.